SPECS/libatomic_ops/libatomic_ops.spec

```diff
@@ -1,14 +1,17 @@
 Summary:        Atomic memory update operations portable implementation
 Name:           libatomic_ops
-Version:        7.6.12
+Version:        7.8.2
 Release:        1%{?dist}
-License:        GPLv2 and MIT
+License:        GPL-2.0-or-later and MIT
 Vendor:         Microsoft Corporation
-Distribution:   Mariner
+Distribution:   Azure Linux
 Group:          Development/Libraries
 URL:            https://github.com/ivmai/libatomic_ops
 Source0:        https://github.com/ivmai/libatomic_ops/releases/download/v%{version}/%{name}-%{version}.tar.gz
 
+BuildRequires:  gcc
```

Comment:
Question: why are these here if they weren't before? Is it a difference in how the versions builds?

Reply:
They're technically not required because they are implied in the defauly build environment (the chroot that contains the toolchain).  However, I think it is bad form to exclude them.

Reply:
Makes sense. Just wanted to double check.